src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -325,11 +341,122 @@ public int getApiVersion()
         }
     }
 
+    public KojiSessionInfo krbLogin()
```

Comment:
OK, I have implemented this. The `prepareRequest()` returns a `String` instead of `KrbLoginRequest` because the `xmlrpcClient.call()` still happens in `KojiClient`. I think this is the way it is supposed to be, as the other component does not have access to `NO_OP_URL_BUILDER` and `STANDARD_REQUEST_MODIFIER`, etc.